input/pagecontent/vitals-write.md

```diff
@@ -40,7 +40,7 @@ Servers **SHALL** support the submission of Observation resources that _are not_
 
 Servers **MAY** support the submission of Observation resources that _are_ the result of a calculation (such as the "US Core Pediatric BMI for Age Observation Profile") and validate against a [US Core Vital Sign Profile](https://build.fhir.org/ig/HL7/US-Core/profiles-and-extensions.html#observation) that corresponds to a version used by the server for vital sign Observation read requests.
 
-Servers **SHALL** respond to supported and valid vital sign Observation creation requests with a status code of `200 OK` and a content location header, or with a status code of `202 Accepted`. If a content location header is provided, the resources **SHALL** be visible in subsequent read API calls and accessible within the system in the same manner as other patient-submitted data. If a content location header is not provided, and the server does not subsequently make the resource accessible in read API calls, the server **SHALL** have a documented and discoverable reason why it was discarded (e.g., a log entry describing rejection during a review workflow or the applicability of a condition in the "Discarding" section below).
+Servers **SHALL** respond to supported and valid vital sign Observation creation requests with a status code of `201 OK` and a `Location` header with a link to the newly created resource. The resources **SHALL** be visible in subsequent read API calls and accessible within the system in the same manner as other patient-submitted data, subject to the system's access control policies, which could potentially prevent a client from immediately reading its writes until some manual review has occurred. If data can be discarded without every becoming visible to the client, the server **SHALL** have a documented policy explaining this behavior and explaining what process can be followed after the fact (e.g., a log entry for administrative users describing rejection during a review workflow, or the applicability of a condition in the "Discarding" section below).
```

Comment:
Think this should be `201 Created`?

Reply:
Also, what about changing "...explaining what process can be followed after the fact..." to something like "...and how to obtain documentation of each application of this policy..."

Reply:
Applied!